src/index.js

```diff
@@ -6,37 +6,23 @@ const validation = {
 
     init() {
 
-        $('[data-validate] input').on('keyup paste', function() {
+        $('[data-validate] input').on('keyup paste blur', function() {
 
-            const $input = $(this);
-
-            if (validateInput($input).isEmpty()) {
-                clearInputError($input);
-            }
+            updateInputError($(this));
```

Comment:
We don't want to update the input error on blur, because then error messages would pop up too soon (e.g. is a user starts typing in an input field, it would display an error after typing every character until it's a valid address)